landing-page/layouts/partials/header.html

```diff
@@ -19,17 +19,11 @@
                     <a class="page-scroll" href="{{ .Site.BaseURL }}docs/latest">Docs</a>
                 </li>
                 <li>
-                    <a class="page-scroll" href="{{ .Site.BaseURL }}docs/latest/getting-started/">Spark</a>
-                </li>
-                <li>
-                    <a class="page-scroll" href="{{ .Site.BaseURL }}docs/latest/flink/">Flink</a>
-                </li>
-                <li>
-                    <a class="page-scroll" href="https://trino.io/docs/current/connector/iceberg.html" target="_blank">Trino</a>
-                </li>
-                <li>
-                    <a class="page-scroll" href="https://prestodb.io/docs/current/connector/iceberg.html" target="_blank">Presto</a>
+                    <a class="page-scroll" href="{{ .Site.BaseURL }}quickstarts/">Quickstarts</a>
                 </li>
+                <!-- <li>
+                    <a class="page-scroll" href="{{ .Site.BaseURL }}concepts/">Concepts</a>
```

Comment:
Keeping the "Concepts" link in the top navbar hidden until we populate that content.